### DISCUSSION TITLE
Add power0 to timer configuration

### DISCUSSION THREAD

**Discussion by nonix** [arendst/Tasmota#13933] [Q&A] [answered]
Hi guys, just an idea, it could be useful to have **power0** when configuring global timers. Perhaps, there is another way how to achieve the same effect of toggling the all configured relays  (e.g. using rule).
![image](https://user-images.githubusercontent.com/1888777/144858298-88bc4e08-f3f3-4d97-ba47-8a3d160f3e9d.png)

**Comment by sfromis** [comment, marked as answer]
You can get the effect by having a rule like `on Clock#Timer==8 do Power0 on endon`

**Reply by nonix** [reply]
Thanks for the answer, the topic was just an idea. I haven't looked into the code on how difficult would it be to add the "0" in the output selection, as this could simplify operation for someone.
Anyway, thank you.

**Reply by sfromis** [reply]
Well, in the larger perspective, only a tiny part of Tasmota features are exposed in the web UI anyway.
